lib/dav4rack/xml_response.rb

```diff
@@ -57,11 +57,8 @@ def render_lock_errors(errors)
     end
 
 
-    def render_lockdiscovery(*args)
```

Comment:
how about `def render_lockdiscovery(**options)` ?

Reply:
The double splat operator used.